pkg/kubectl/cmd/set/set_env.go

```diff
@@ -74,10 +74,10 @@ var (
 	  kubectl set env rc --all ENV=prod
 
 	  # Import environment from a secret
-	  kubectl set env --from=secret/mysecret dc/myapp
+	  kubectl set env --from=secret/mysecret deployment/myapp
```

Comment:
why update this?

Reply:
kubectl can not recognize dc as abbreviation of  deployment config

Reply:
@zjj2wry WDYT

Reply:
yes, you are right